pkg/kube/inject/inject_test.go

```diff
@@ -121,6 +122,19 @@ func TestIntoResourceFile(t *testing.T) {
 			readinessFailureThreshold:    DefaultReadinessFailureThreshold,
 			enableCni:                    true,
 		},
+		// verify cni network
```

Comment:
I'm curious, in the case above, shouldn't the file `hello.yaml.cni.injected` contain the annotation with the default value?

Reply:
No, the default values are not consumed in these tests